vsetup.sh

```diff
@@ -94,6 +94,8 @@ function env_all_pre ()
 function env_all_pre_centos () {
     say "Running yum makecache fast..."
     yum makecache fast
+    say "Installing lsb_release..."
+    yum yum install -y redhat-lsb-core
```

Comment:
You're right, I just went with what maede things work for me. And then even pushed the wrong commit with a typo. I'll fix it.